webpack/webpack5-localization-plugin/src/test/NonHashedNonLocalizedAssets.test.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+jest.disableAutomock();
+import { promisify } from 'util';
+
+import webpack, { type Stats } from 'webpack';
+import { Volume } from 'memfs/lib/volume';
+
+import { TrueHashPlugin } from '../TrueHashPlugin';
+import { MemFSPlugin } from './MemFSPlugin';
+
+async function testNonLocalizedInner(minimize: boolean): Promise<void> {
+  const memoryFileSystem: Volume = new Volume();
+  memoryFileSystem.fromJSON(
+    {
+      '/package.json': '{}',
+      '/entrySingleChunk.js': `console.log("Do stuff");import(/* webpackChunkName: 'async1' */ './async1.js').then(mod => mod.foo());`,
+      '/entryTwoChunks.js': `console.log("Do stuff");import(/* webpackChunkName: 'async1' */ './async1.js').then(mod => mod.foo());import(/* webpackChunkName: 'async2' */ './async2.js').then(mod => mod.foo());`,
+      '/async1.js': `export function foo() { console.log('foo1'); }`,
+      '/async2.js': `export function foo() { console.log('foo2'); }`
+    },
+    '/src'
+  );
+
+  const trueHashPlugin: TrueHashPlugin = new TrueHashPlugin();
+
+  const compiler: webpack.Compiler = webpack({
+    entry: {
+      mainSingleChunk: '/entrySingleChunk.js',
+      mainTwoChunks: '/entryTwoChunks.js'
+    },
+    output: {
+      path: '/release',
+      filename: '[name]-[locale].js'
```

Comment:
```suggestion
      filename: '[name].js'
```
To make it obvious that there is no localization plugin present.